account-integrations/safe/test/hardhat/SafeBlsPlugin.test.ts

```diff
@@ -140,19 +133,19 @@ describe("SafeBlsPlugin", () => {
       maxPriorityFeePerGas,
       paymasterAndData: "0x",
       signature: "",
-    };
+    } satisfies UserOperationStruct;
```

Comment:
This is a good case study demonstrating the utility of the new `satisfies` keyword. If it's unclear, I recommend [this video](https://www.youtube.com/watch?v=49gHWuepxxE) for background, and taking a look what happens when you modify this statement back to using `: UserOperationStruct`.